py/examples/theme_generator.py

```diff
@@ -42,9 +42,24 @@ def get_contrast(color1: str, color2: str, q: Q, min_contrast=4.5):
     darkest = min(lum1, lum2)
     contrast = (brightest + 0.05) / (darkest + 0.05)
     if contrast < min_contrast:
-        return ui.message_bar(type='error', text=f'Improve contrast between **{color1}** and **{color2}**.')
+        return False
     else:
-        return ui.message_bar(type='success', text=f'Contrast between **{color1}** and **{color2}** is great!')
+        return True
+    
+def update_contrast_check(color1: str, color2: str, q:Q, min_contrast=4.5):
```

Comment:
I suppose this function deprecates the original `get_contrast`. If so, `get_contrast` should be removed as it's not used anymore.

Reply:
Yep, a leftover. Thanks.